2996_reflection/reflection.md

```diff
@@ -1823,6 +1819,60 @@ Change the grammar for `$operator-or-punctuator$` in paragraph 1 of [lex.operato
 ```
 :::
 
+### [basic.types.general]
+
+Change the first sentence in paragraph 9 of [basic.types.general]{.sref} as follows:
+
+::: bq
+[9]{.pnum} Arithmetic types (6.8.2), enumeration types, pointer types, pointer-to-member types (6.8.4),[ `std::meta::info`,]{.addu} `std::nullptr_t`, and cv-qualified (6.8.5) versions of these types are collectively called scalar types.
+:::
+
+Add a new paragraph at the end of [basic.types.general]{.sref} as follows:
+
+::: bq
+::: addu
+
+[*]{.pnum} A *consteval-only* is one of the following:
+
+  - `std::meta::info`, or
+  - a pointer or reference to a consteval-only type, or
+  - an (possibly multi-dimensional) array of a consteval-only type, or
+  - a type pointer-to-member of class C of type M where C and/or M is a consteval-only type, or
+  - a function type with a consteval-only return type or a consteval-only parameter type, or
+  - a class type with a consteval-only base class type or consteval-only nonstatic data member type.
```

Comment:
```suggestion
  - a class type with a consteval-only base class type or consteval-only non-static data member type.
```

Reply:
(I'm still used to the style guide Addison-Wesley required, which prohibited the hyphen after `non` unless pronunciation became ambiguous.)

Reply:
Yeah but we always spell it non-static 🤷